scraper_place/extraction.py

```diff
@@ -24,10 +24,10 @@ def extract():
         aws_access_key_id=CONFIG_S3['aws_access_key_id'],
         aws_secret_access_key=CONFIG_S3['aws_secret_access_key'],
         region_name=CONFIG_S3['region_name'],
-    ).resource('s3')
+    ).resource('s3', endpoint_url=CONFIG_S3['endpoint_url'])
```

Comment:
```suggestion
    ).resource('s3', endpoint_url=CONFIG_S3.get('endpoint_url'))
```
pour ne pas casser la prod ?

Reply:
J'ai testé, en effet le .get() donne `None` en fallback par défaut ce qui est également la valeur par défaut de endpoint_url. Ca devrait donc bien se passer a priori...